lib/index.js

```diff
@@ -155,4 +163,22 @@ exports.register = (server, options) => {
     }
 };
 
-exports.name = NAME;
+exports.addAxiosRoutePreHook = {
+    assign: 'axios',
+    method: (request) => {
+        const factory = get(request, ['server', 'plugins', NAME, 'axiosFactory']);
+        const { traceUUIDHeader, traceDepthHeader, config } = get(request, ['server', 'plugins', NAME, 'axiosConfig'], {});
+
+        // provide axios client to request
+        const axiosConfig = Hoek.applyToDefaults(config, {
+            headers: {
+                common: {
+                    [traceUUIDHeader]: get(request, ['plugins', NAME, traceUUIDHeader]),
+                    [traceDepthHeader]: get(request, ['plugins', NAME, traceDepthHeader])
```

Comment:
Who is responsible for incrementing the depthHeader?  I was expecting it to be incremented here so all new calls using axios would be using the new depth.

Reply:
This library handles the incrementing internally.

The `incrementDepth` method is implemented here: https://github.com/GoodwayGroup/lib-hapi-good-tracer/blob/master/lib/index.js#L26

It is called in the `onRequest` event that is hooked into here: https://github.com/GoodwayGroup/lib-hapi-good-tracer/blob/master/lib/index.js#L106-L109

Reply:
That is to say this library implements the features according to the spec laid out [in coding-standard](https://github.com/GoodwayGroup/coding-standards/blob/master/service-request-tracing.md)